src/app/ReadClient.h

```diff
@@ -124,7 +124,7 @@ class ReadClient : public Messaging::ExchangeDelegate
          * This function will:
          *      - Always be called exactly *once* for a given WriteClient instance.
          *      - Be called even in error circumstances.
-         *      - Only be called after a successful call to SendWriteRequest as been made.
+         *      - Only be called after a successful call to SendWriteRequestMessage as been made.
```

Comment:
I don't understand what `SendWriteRequestMessage` has to do with `ReadClient`.

Reply:
sure, rename it back.